drivers/accesskit/dynwrappers/accesskit-dll_wrap.c

```diff
@@ -0,0 +1,4319 @@
+#undef accesskit_node_bounds
+#undef accesskit_node_set_bounds
+#undef accesskit_node_clear_bounds
+#undef accesskit_node_text_selection
+#undef accesskit_node_set_text_selection
+#undef accesskit_node_clear_text_selection
+#undef accesskit_custom_action_new
+#undef accesskit_custom_actions_free
+#undef accesskit_node_custom_actions
+#undef accesskit_node_set_custom_actions
+#undef accesskit_node_push_custom_action
+#undef accesskit_node_clear_custom_actions
+#undef accesskit_node_new
+#undef accesskit_node_free
+#undef accesskit_tree_new
+#undef accesskit_tree_free
+#undef accesskit_tree_get_toolkit_name
+#undef accesskit_tree_set_toolkit_name
+#undef accesskit_tree_clear_toolkit_name
+#undef accesskit_tree_get_toolkit_version
+#undef accesskit_tree_set_toolkit_version
+#undef accesskit_tree_clear_toolkit_version
+#undef accesskit_tree_update_with_focus
+#undef accesskit_tree_update_with_capacity_and_focus
+#undef accesskit_tree_update_free
+#undef accesskit_tree_update_push_node
+#undef accesskit_tree_update_set_tree
+#undef accesskit_tree_update_clear_tree
+#undef accesskit_tree_update_set_focus
+#undef accesskit_action_request_free
+#undef accesskit_affine_identity
+#undef accesskit_affine_flip_y
+#undef accesskit_affine_flip_x
+#undef accesskit_affine_scale
+#undef accesskit_affine_scale_non_uniform
+#undef accesskit_affine_translate
+#undef accesskit_affine_map_unit_square
+#undef accesskit_affine_determinant
+#undef accesskit_affine_inverse
+#undef accesskit_affine_transform_rect_bbox
+#undef accesskit_affine_is_finite
+#undef accesskit_affine_is_nan
+#undef accesskit_point_to_vec2
+#undef accesskit_rect_from_points
+#undef accesskit_rect_from_origin_size
+#undef accesskit_rect_with_origin
+#undef accesskit_rect_with_size
+#undef accesskit_rect_width
+#undef accesskit_rect_height
+#undef accesskit_rect_min_x
+#undef accesskit_rect_max_x
+#undef accesskit_rect_min_y
+#undef accesskit_rect_max_y
+#undef accesskit_rect_origin
+#undef accesskit_rect_size
+#undef accesskit_rect_abs
+#undef accesskit_rect_area
+#undef accesskit_rect_is_empty
+#undef accesskit_rect_contains
+#undef accesskit_rect_union
+#undef accesskit_rect_union_pt
+#undef accesskit_rect_intersect
+#undef accesskit_size_to_vec2
+#undef accesskit_vec2_to_point
+#undef accesskit_vec2_to_size
+#undef accesskit_windows_queued_events_raise
+#undef accesskit_windows_adapter_new
+#undef accesskit_windows_adapter_free
+#undef accesskit_windows_adapter_update_if_active
+#undef accesskit_windows_adapter_update_window_focus_state
+#undef accesskit_windows_adapter_handle_wm_getobject
+#undef accesskit_windows_subclassing_adapter_new
+#undef accesskit_windows_subclassing_adapter_free
+#undef accesskit_windows_subclassing_adapter_update_if_active
+
+#define RTLD_LAZY 0
+
+char *dl_error = NULL;
+char dl_error_text[2048];
+
+void *dlsym(void *p_handle, const char *p_name) {
+  dl_error = NULL;
+
+  HMODULE hmodule = (HMODULE)p_handle;
+  void *f = GetProcAddress(hmodule, p_name);
```

Comment:
Getting a `C4152` warning here without a cast
```suggestion
  void *f = (void *)GetProcAddress(hmodule, p_name);
```